go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/filecoin-project/go-address v0.0.5
 	github.com/filecoin-project/go-dagaggregator-unixfs v0.2.0
 	github.com/filecoin-project/go-indexer-core v0.2.15
-	github.com/filecoin-project/go-legs v0.3.14
+	github.com/filecoin-project/go-legs v0.3.15-0.20220519133545-bcb09a8350f0
```

Comment:
To be replaced with a concrete version once https://github.com/filecoin-project/go-legs/pull/126 is merged.